coralproject/talk
# Configurable Limits

## What does this PR do?

- Adds support for configuring the stream load limits.
- Significant enhancements to action summary loader (often removing the need to call the database).

## How do I test this PR?

Set:
- `TALK_ASSET_COMMENTS_LOAD_DEPTH=20` - How many comments display on initial load
- `TALK_REPLY_COMMENTS_LOAD_DEPTH=20` - How many replies display on initial load
- `TALK_ADDTL_COMMENTS_ON_LOAD_MORE=20` - How many comment display once the reader clicks "View More Comments" 

1. Build assets with `yarn build`
2. Create 20+ comments.
3. Refresh stream.
4. Notice that you (on initial load) can scroll to see at least 20 comments.
